nix/ratelimit/.gitignore

```diff
@@ -0,0 +1 @@
+result-bin
```

Comment:
I haven't seen `.gitignore` placed anywhere else but in the top-level repository directory. Why a separate file here instead of putting that single line in the top-level `.gitignore`?

Reply:
Good find; it should have gone in the top-level .gitignore. I've removed it now, as it was only needed in an older version of default.nix I was writing, but not in the version pushed in the PR.